scripts/menu-script/run-tfplugindocs.sh

```diff
@@ -0,0 +1,4 @@
+ curl -L https://github.com/hashicorp/terraform-plugin-docs/releases/download/v0.13.0/tfplugindocs_0.13.0_darwin_arm64.zip  --output /usr/local/bin/tfplugindocs.zip
```

Comment:
What would happen if there is a new release of tfplugindocs and the version is different?